### DISCUSSION TITLE
How can I custom definition my loader

### DISCUSSION THREAD

**Discussion by chenxiaofie** [webpack/webpack#17010] [Q&A] [answered]
Help me pleas

**Comment by shadowgxq** [comment, marked as answer]
Install any dependencies your loader needs, if necessary. For example, if your loader uses a library like lodash, you can install it with npm:
```npm install lodash --save-dev```
In your Webpack configuration file, specify the path to your loader file and any options it needs in the module.rules array.
r example, let's say your Webpack configuration file is called webpack.config.js. You can add the following rule to apply your replace-foo-loader to all .js files:
```module.exports = {
  // ...other configuration options...
  module: {
    rules: [
      {
        test: /\.js$/,
        use: [
          {
            loader: path.resolve(__dirname, 'replace-foo-loader.js'),
            options: {
              // any options your loader needs
            }
          }
        ]
      }
    ]
  }
};

```
